scss/sections/_trade.scss

```diff
@@ -2,37 +2,121 @@
     background-color: rgb(245, 245, 245);
     height: 1480px;
     color: $primary;
-    .primary_header {
-        display: flex;
-        flex-direction: column;
-        align-items: center;
+    .trade_header {
+        display: flex;;
         text-align: center;
-        background-image: url('../../img/background/section-trade-XL.png');
-        height: 450px;
-        margin: 100px 0;
         p {
             font-size: 40px;
             font-weight: 600;
         }
+    }
+    .primary_header {
+        flex-direction: column;
+        background-image: url('../../img/background/section-trade-XL.png');
+        height: 350px;
+        margin: 100px 0 0 0;
         span {
-            margin-top: 40px;
+            margin-top: 30px;
             color: $greycolor;
         }
     }
     .calculator {
         position: relative;
-        bottom: 240px;
+        bottom: 140px;
         margin: 0 auto;
         height: 350px;
-        width: 70vw;
+        width: 70%;
         background-color: #fff;
+        color: #303030;
         border-radius: 20px;
+        .calculator_in {
+            display: flex;
+            flex-direction: row;
+            padding-top: 30px;
+            font-size: 20px;
+            input {
+                margin: 30px 12%;
+                height: 2rem;
+                font-size: 18px;
+                border-top-style: none;
+                border-left-style: none;
+                border-right-style: none;
+                outline: none;
+            }
+        }
+        input:valid ~ .input_text,
+        input:focus ~ .input_text {
+            opacity: 0;
+        }
+        .input_text {
+            position: relative;
+            bottom: 60px;
+            margin: 0 12%;
+            transition: 0.5s;
+        }
+        .left_in {
+            width: 50%;
+            .text_in {
+                width: 80%;
+            }
+        }
+        button {
+            margin: 5px 5% 5px 10%;
+            border: none;
+            color: $primary;
+            width: 140px;
+        }
     }
     .secondary_header {
-        display: flex;
-        align-items: center;
-        text-align: center;
-        margin: 100px 0;
-
+        justify-content: center;
+        p {
+            color: black;
+        }
     }
-}
\ No newline at end of file
+}
+
+// .left_in {
```

Comment:
NEVER left commented code when you do PR.... PR should be complete code of all changes ....